tests/Feature/SaleTest.php

```diff
@@ -213,4 +213,63 @@ public function test_sales_can_be_updated()
         ]);
         $this->assertEquals("69.99", $updatedData['data']->price);
     }
+
+        /**
+     *
+     * @test
+     * @return void
+     */
+    public function test_sales_can_be_shown()
+    {
+
+        // Create a sale
+        $sale = Sale::create([
+            'description' => "This items will be a test sale",
+            'client_name' => "test name",
+            'status' => "test status",
+            'payment_type' => "test payment",
+            'card_number' => "12341234123412349876987698769876",
```

Comment:
Good work though some of the values you set for the Sale fields are invalid.
- `status` field can only contain : pending, shipped ,paid, or cancelled (Case-sensitive)
- `payment_type` can only contain : Visa or Master Card (Case-sensitive)
- `card_number` can only contain : 16 numeric digits

Reply:
Example :
'status' => "pending",
'payment_type' => "Visa",
'card_number' => "1234123412341234",

Reply:
Nice catch! Changes were made!